blog/_posts/2018-07-19-team-sunshine.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Team ☀️ 'Sunshine' on RGSoC 2018!
+layout: post
+created_at: Thu Jul 19 2018
+permalink: blog/2018-07-19-team-sunshine
+current: blog
+author: Sujin Lee
+twitter: sujinleeme
+categories:
+- student-posts
+---
+
+# Who we are
+![team sunshine photo](/img/blog/2018/2018-07-19-team-sunshine.jpg "team sunshine photo")
```

Comment:
Hi @sujinleeme ! Like I said in my previous review, you also need to add image credits for all images in teh file. You seem to have missed it. :)